include/umf/memory_pool_ops.h

```diff
@@ -17,6 +17,26 @@
 extern "C" {
 #endif
 
+/// @brief This structure comprises function pointers used by corresponding umfMemoryPool calls.
+typedef struct umf_memory_pool_ext_ops_t {
+    ///
+    /// @brief Control operation for the memory provider.
```

Comment:
There is a mix of memory pool naming and memory provider comments in this struct. Probably you meant to describe memory pool in comments?

Reply:
Done